RecoLocalCalo/HcalRecProducers/src/HBHEPhase1Reconstructor.cc

```diff
@@ -518,17 +528,17 @@ void HBHEPhase1Reconstructor::processData(const Collection& coll,
             const float t = getTDCTimeFromSample(s);
             const float dfc = getDifferentialChargeGain(*channelCoder, *shape, adc,
                                                         capid, channelInfo->hasTimeInfo());
-            channelInfo->setSample(ts, adc, dfc, rawCharge,
+            channelInfo->setSample(ts-shiftOneTS, adc, dfc, rawCharge,
                                    pedestal, pedestalWidth,
                                    gain, gainWidth, t);
-            if (ts == soi)
+            if ((ts-shiftOneTS) == (soi-shiftOneTS))
                 soiCapid = capid;
         }
 
         // Fill the overall channel info items
         const int pulseShapeID = param_ts->pulseShapeID();
         const std::pair<bool,bool> hwerr = findHWErrors(frame, maxTS);
-        channelInfo->setChannelInfo(cell, pulseShapeID, maxTS, soi, soiCapid,
+        channelInfo->setChannelInfo(cell, pulseShapeID, maxTS-2*shiftOneTS, soi-shiftOneTS, soiCapid,
```

Comment:
similarly ```maxFitTS``` and ```fitSoi``` may be easier to read than these explicitly shifted versions.

Reply:
I updated the code with `maxFitTS` and `fitSoi` following your suggestion.